posix.mak

```diff
@@ -21,6 +21,7 @@ clean:
 	$(RM) tags
 
 test:
+	$(QUIET)$(MAKE) -C src -f posix.mak unittest
```

Comment:
It looks like you don't have to modify `src/posix.mak`, if you simply replace here `unittest` with `ENABLE_DEBUG=1`.
(For reference, ,`ENABLE_DEBUG` does the following:
```Makefile
ifdef ENABLE_DEBUG
 CXXFLAGS += -g -g3 -DDEBUG=1 -DUNITTEST
 DFLAGS += -g -debug -unittest
````
).

I think going with this option, instead of introducing a new target there would be simpler/better. What do you think?

Reply:
I prefer a separate target because then it's a lot easier to run the `unittest` target with the currently compiled `dmd` binary. Imho it's a common concept to use a different binary for the unittest executable and deviating from this might lead to confusion or annoyance down the road.

Reply:
Alright, you make a good point.